source/features/conversation-navigation.tsx

```diff
@@ -0,0 +1,163 @@
+	}
+}
+
+type Conversation = {
+	cursor: number;
+	number: number;
+	url: string;
+};
+
+const fetchConversationList = cache.function(async (query: string, page: number): Promise<Conversation[]> => {
+	// When fetching a page of conversations we want to also fetch last item from previous page, and first item from next page.
+	// This is needed for cases when conversation is first/last in list, and we need to know if items exist on previous/next page.
+	// To take into consideration also, that when we are on first item on first page, we don't have a previous item, and we don't want to fetch using a negative cursor.
+	const {search} = await api.v4(`
+		search(
+			type: ISSUE,
+			first: ${page === 1 ? ITEMS_PER_PAGE + 1 : ITEMS_PER_PAGE + 2},
+			after: "${btoa(`cursor:${page === 1 ? 0 : ((page - 1) * ITEMS_PER_PAGE) - 1}`)}",
+			query: "${query}"
+		) {
+			edges {
+				cursor
+				node {
+					... on Issue { number, url }
+					... on PullRequest { number, url }
+				}
+			}
+		}
+	`);
+
+	return search.edges.map((edge: any) => ({
+		...edge.node,
+		// GitHub GraphQL cursor is an offset number starting at 1, with prefix `cursor:` and converted to base64.
+		// e.g. offset 42 is `cursor:42` and converted to base64 is `Y3Vyc29yOjQy`
+		// Always query considering items offsets starting at 1, otherwise unexpected results may happen.
+		cursor: looseParseInt(atob(edge.cursor))
+	}));
+},
+{
+	cacheKey: ([query, page]) => `${__filebasename}:${query}/${page}`,
+	maxAge: 1 / 24
+});
+
+// Current default number of items in a conversation list
+const ITEMS_PER_PAGE = 25;
+
+async function getConversationList(conversationNumber: number) {
+	const listQuery = getConversationListQuery();
+	const query = listQuery.get();
+	let page = Number(listQuery.searchParams.get('page') ?? 1);
+
+	const list = await fetchConversationList(query, page);
+	const conversation = list.find(item => item.number === conversationNumber)!;
+
+	if (conversation.cursor === (page - 1) * ITEMS_PER_PAGE) {
+		// `currentConversation` is from previous page, we need to fetch previous page items
+		page -= 1;
+	} else if (conversation.cursor === (page * ITEMS_PER_PAGE) + 1) {
+		// `currentConversation` is from next page, we need to fetch next page items
+		page += 1;
+	} else {
+		return {query, page, list};
+	}
+
+	return {
+		query,
+		page,
+		list: await fetchConversationList(query, page)
+	};
+}
+
+const noListQuery = (): boolean => !(
+	// Allowed also from global conversation list
+	pageDetect.isConversationList(new URL(document.referrer)) ||
+	new URL(location.href).searchParams.has('q')
+);
+
+function getConversationListQuery(): SearchQuery {
+	const referrerUrl = new URL(document.referrer);
+	const url = pageDetect.isConversationList(referrerUrl) ? referrerUrl : location;
+
+	if (pageDetect.isRepoConversationList(url)) {
+		return new SearchQuery(url).add(`repo:${getRepoURL()}`);
+	}
+
+	return new SearchQuery(url);
+}
+
+void features.add({
+	id: __filebasename
```

Comment:
The point of using strict types here is to make it painfully obvious that they need to be set. If we use junk data (`description: '', screenshot: false`) we defeat its purpose, that's why I removed them in https://github.com/sindresorhus/refined-github/pull/3467/commits/be54b915ae65e829c2db121d13645113c15512fb

Reply:
Added a more descriptive description, and screenshot to false

Reply:
Why did you set false again? This feature needs a screenshot